### DISCUSSION TITLE
Expose JMX remote port 1099

### DISCUSSION THREAD

**Discussion by ranajoy1998** [arkmq-org/arkmq-org-broker-operator#973] [Q&A]
How can I expose the JMX remote port 1099 from the artemis-broker-container? I tried enabling the connector in management.xml and building image but if I describe the pod in kubernetes, container is only listening on 8161. I need to enable this port so that I can monitor the cluster from Datadog. I didn't find anything in the CRD, can someone please guide! Thanks in advance.

**Comment by brusdev** [comment]
You coudl use resource templates to apply [Custom Modifications via a Strategic Merge Patch](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#custom-modifications-via-a-strategic-merge-patch) however I would suggest you to [Enable broker's metrics plugin](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#enable-brokers-metrics-plugin) and use the [Datalog OpenMetrics integration](https://docs.datadoghq.com/integrations/openmetrics/).
